pandas/tests/io/formats/test_to_excel.py

```diff
@@ -200,5 +210,222 @@ def test_css_to_excel_multiple():
      {'font': {'italic': True}}),
 ])
 def test_css_to_excel_inherited(css, inherited, expected):
+    pytest.importorskip('cssdecl')
     convert = CSSToExcelConverter(inherited)
     assert expected == convert(css)
+
+
+@pytest.fixture
+def styled_dataframe():
+    def style(df):
+        # XXX: RGB colors not supported in xlwt
+        return DataFrame([['font-weight: bold', '', ''],
+                          ['', 'color: blue', ''],
+                          ['', '', 'text-decoration: underline'],
+                          ['border-style: solid', '', ''],
+                          ['', 'font-style: italic', ''],
+                          ['', '', 'text-align: right'],
+                          ['background-color: red', '', ''],
+                          ['', '', ''],
+                          ['', '', ''],
+                          ['', '', '']],
+                         index=df.index, columns=df.columns)
+
+    pytest.importorskip('jinja2')
```

Comment:
are you asking if the fixture also needs cssdecl to be imported? no, since we allow for a custom `style_converter`.